<?xml version="1.0" ?>
<sdf version="1.6">
  <world name="default">
    <gravity>0 0 -9.8</gravity>

    <include>
      <uri>model://ground_plane</uri>
    </include>
    <include>
      <uri>model://sun</uri>
    </include>

    <include>
      <uri>model://trisphere_cycle</uri>
      <name>trisphere_cycle_slip</name>
      <pose>0 2 0  0 0 0</pose>
      <plugin name="wheel_slip" filename="libgazebo_ros_wheel_slip.so">
        <ros>
          <namespace>trisphere_cycle_slip</namespace>
        </ros>
        <wheel link_name="wheel_front">
          <slip_compliance_lateral>1</slip_compliance_lateral>
          <slip_compliance_longitudinal>1</slip_compliance_longitudinal>
          <wheel_normal_force>32</wheel_normal_force>
        </wheel>
        <wheel link_name="wheel_rear_left">
          <slip_compliance_lateral>1</slip_compliance_lateral>
          <slip_compliance_longitudinal>1</slip_compliance_longitudinal>
          <wheel_normal_force>32</wheel_normal_force>
        </wheel>
        <wheel link_name="wheel_rear_right">
          <slip_compliance_lateral>1</slip_compliance_lateral>
          <slip_compliance_longitudinal>1</slip_compliance_longitudinal>
          <wheel_normal_force>32</wheel_normal_force>
        </wheel>
      </plugin>
    </include>

    <gui fullscreen='0'>
      <camera name='user_camera'>
        <pose>1.5 -4 2.5  0 0.5 1.6</pose>
        <view_controller>orbit</view_controller>
      </camera>
    </gui>
  </world>
</sdf>
